sirepo/util.py

```diff
@@ -447,6 +463,10 @@ def to_comma_delimited_string(arr):
     return ",".join([str(x) for x in arr])
 
 
+def unique_key():
```

Comment:
This deserves a comment since it's been more widely published. Clients can assume it is a "reasonably sized" web- and file-system safe token that is randomly generated from enough bits that is globally unique. It is our version of UUID4 without the structure. We might make some guarantee, e.g. less than 128 characters, so it can be stored in a database with some type of structure.

Reply:
97e3e0c25